### ISSUE TITLE
Common.js Module is missing

### ISSUE THREAD

**Issue by pahaz** [iddan/react-spreadsheet#66]
I use the next.js. And try to run the example

```
/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/node_modules/react-spreadsheet/dist/SpreadsheetStateProvider.js:1
import _objectWithoutProperties from "@babel/runtime/helpers/esm/objectWithoutProperties";
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (internal/modules/cjs/loader.js:1070:16)
    at Module._compile (internal/modules/cjs/loader.js:1120:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.react-spreadsheet (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:10383:18)
    at __webpack_require__ (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:29:31)
    at Module../pages/index.jsx (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:10057:75)
    at __webpack_require__ (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:29:31)
    at Object.4 (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:10206:18)
    at __webpack_require__ (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:29:31)
    at /Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:104:18
    at Object.<anonymous> (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/apps/_example05app/.next/server/static/development/pages/index.js:107:10)
    at Module._compile (internal/modules/cjs/loader.js:1156:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1176:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Module.require (internal/modules/cjs/loader.js:1042:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.requirePage (/Users/pahaz/Code/node-docker-compose-dev/my-keystone-app/node_modules/next/dist/next-server/server/require.js:46:12)
```

**Comment by r3wt** [comment]
It is caused because this project only exports an es module, but provides no commonjs module. so it has incompatibility with commonjs projects. i am having the same problem in my project, it runs fine in build and a dev mode, but crashes when running tests in jest. i am not sure of fix, other than changing the source to export both module types, which is beyond my control. if i find a fix for myself i will pass it along.

**Comment by r3wt** [comment]
For jest, the workaround i used was:

1. create .babelrc.json file with following content:
```
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "modules": "commonjs"
      }
    ]
  ]
}
```

2. create jest wrapper like jest-wrapper.js

```js
module.exports = require("babel-jest").createTransformer({
  rootMode: "upward",
});
```

3. modify package.json to configure jest to ignore the modules we want to transform, and instruct jest to use our custom jest wrapper:

package.json changes:
```
{
...
"jest":{
"transform": {
      "^.+\\.js?$": "./jest-wrapper.js"
    },
    "transformIgnorePatterns": [
      "/node_modules/(?!react-spreadsheet|@babel).+\\.js$"
    ],
}
...
}
```

**Comment by iddan** [comment]
How can I solve this issue? @r3wt

**Comment by iddan** [comment]
Fixed in 0.5.0.
